cmd/tealdbg/local.go

```diff
@@ -128,6 +128,16 @@ func balanceRecordsFromParams(dp *DebugParams) (records []basics.BalanceRecord,
 		records = append(records, record)
 	}
 
+	// if conversion failed report all intermediate decoding errors
+	if err != nil {
+		if err1 != nil {
+			log.Printf("Decoding as JSON txn failed: %s", err1.Error())
```

Comment:
you're not decoding a txn here, you're decoding a balance record. 
i.e.
"Decoding blob as a JSON encoded single balance record failed : %s"

Reply:
Fixed, thanks